broker-util/oo-admin-chk

```diff
@@ -171,7 +171,8 @@ node_hash.each do |gear_uuid, gear_info|
     if !datastore_has_gear?(gear_uuid)
       puts "#{gear_uuid}...FAIL" if $verbose
       print_message "Gear #{gear_uuid} exists on node #{gear_info[0]} (uid: #{gear_info[1]}) but does not exist in mongo database"
-      print_message "Please see https://access.redhat.com/solutions/1171163 for more information."
+      print_message "To fix this issue, remove the gear by using the oo-devel-node command:"
```

Comment:
The `oo-admin-chk` command is on the broker, and the `oo-devel-node` command is on the node, right? If the user runs a command on the broker, and the output of that first command just says to run a second command but doesn't say where to run it, the user is likely to try to run the second command on the same host.